test/index.test.js

```diff
@@ -81,4 +82,17 @@ describe('percySnapshot', () => {
     expect(() => percySnapshot())
       .toThrowError('The WebdriverIO `browser` object is required.');
   });
+
+  it('throws error for percy on automate session', async () => {
+    spyOn(percySnapshot, 'isPercyEnabled').and.returnValue(Promise.resolve(true));
```

Comment:
```suggestion
```

we don't need this, as percy is enabled.

Reply:
So actually i want to set utils.percy.type to "automate" now during normal healthcheck call i.e isPercyEnabled call it is set to "web" since we run test using percy-web token.

Reply:
but you're not testing `isPercyEnabled` anyways right? you're mocking the function so doesn't matter which token type it is.

Reply:
So basically flow is like this

- percySnapshot()
-   healthcheck call
-     sets utils.percy.type to web/automate

Now i can't modify utils.percy.type = "automate" value before calling percySnapshot since it will be overridden to web inside healthcheck. after calling percySnapshot updating value also doesn't makes sense. basically we are running test using web token so utils.percy.type will always be web just to test behaviour when it is automate we have mock it.

Reply:
so I observed that the dependency of the sdk-utils is quite old here, so thats the reason its working since there was no PoA CLI code present.